src/workspace.coffee

```diff
@@ -612,16 +612,19 @@ class Workspace extends Model
 
   # Increase the editor font size by 1px.
   increaseFontSize: ->
+    @originalFontSize ?= atom.config.get("editor.fontSize")
```

Comment:
I think what might do here is emit a `did-change-font-size` event here and in settings-view.